vispy/gloo/glir.py

```diff
@@ -894,13 +898,57 @@ def set_wrapping(self, wrapping):
             gl.glTexParameterf(self._target, GL_TEXTURE_WRAP_R, wrapping[0])
         gl.glTexParameterf(self._target, gl.GL_TEXTURE_WRAP_S, wrapping[-2])
         gl.glTexParameterf(self._target, gl.GL_TEXTURE_WRAP_T, wrapping[-1])
-    
+
     def set_interpolation(self, min, mag):
         self.activate()
         min, mag = as_enum(min), as_enum(mag)
         gl.glTexParameterf(self._target, gl.GL_TEXTURE_MIN_FILTER, min)
         gl.glTexParameterf(self._target, gl.GL_TEXTURE_MAG_FILTER, mag)
 
+# these should be auto generated in _constants.py. But that doesn't seem 
+# to be happening. TODO - figure out why the C parser in (createglapi.py)
+# is not extracting these constanst out.
+# found the constant value at:
```

Comment:
I found this to be _very icky_. Is there a neater way that we could generate these constants? I'm ready to write code to automate this if there's any non-insanely complex way to automate this